features/staging_commands.feature

```diff
@@ -88,7 +88,19 @@ Feature: Enable command behaviour when attached to an UA staging subscription
         """
         1
         """
-
+        When I run `ua disable fips --assume-yes` with sudo
+        Then stdout matches regexp:
+            """
+            This will disable access to certified FIPS packages.
```

Comment:
When running locally, the test fails because the sentence:
```
This will disable access to certified FIPS packages.
```

is not being displayed during the disable operation

Reply:
Ahh right, that output is only presented without the --assum-yes option.